extensions/cli/bigtable-embed/src/main/java/org/locationtech/geowave/datastore/bigtable/cli/RunBigtableEmulatorOptions.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2013-2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file distributed with this work for additional information regarding copyright
+ * ownership. All rights reserved. This program and the accompanying materials are made available
+ * under the terms of the Apache License, Version 2.0 which accompanies this distribution and is
+ * available at http://www.apache.org/licenses/LICENSE-2.0.txt
+ */
+package org.locationtech.geowave.datastore.bigtable.cli;
+
+import java.io.IOException;
+import com.beust.jcommander.Parameter;
+
+public class RunBigtableEmulatorOptions {
+  @Parameter(names = {"--directory", "-d"}, description = "The directory to use for BigTable")
+  private String directory = null;
+
+  @Parameter(names = {"--url", "-u"}, description = "The url location to download BigTable")
+  private String url = "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads";
+  
+  @Parameter(names = {"--sdk", "-s"}, description = "The name of the BigTable SDK")
+  private String sdk = "google-cloud-sdk-183.0.0-linux-x86_64.tar.gz";
+  
+  @Parameter(names = {"--port", "-p"}, description = "The port the emulator will run on")
+  private String port = "127.0.0.1:8086";
```

Comment:
Since this is always run locally, change this to be a port only and append the address when actually starting the server.